pyvista/core/_typing_core/_aliases.py

```diff
@@ -79,3 +80,5 @@ class BoundsTuple(NamedTuple):
 
 # Undocumented alias - should be expanded in docs
 _ArrayLikeOrScalar = Union[NumberType, ArrayLike[NumberType]]
+
+InteractionEventType = Literal['end', 'start', 'always'] | _vtk.vtkCommand.EventIds
```

Comment:
Use old syntax for types used at runtime 
```suggestion
InteractionEventType = Union[Literal['end', 'start', 'always'], _vtk.vtkCommand.EventIds]
```